awssample.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.27"
+    }
+  }
+  required_version = ">= 0.14.9"
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "us-west-1"
+}
+
+resource "aws_instance" "app_seraver" {
```

Comment:
Issue - EC2 instances should disable IMDS or require IMDSv2 as this can be related to the weaponization phase of kill chain 
 category - Infrastructure Security 
 severity - MEDIUM